config/bidderinfo.go

```diff
@@ -243,10 +252,49 @@ func processBidderInfos(reader InfoReader, normalizeBidderName func(string) (ope
 				return nil, fmt.Errorf("error parsing config for bidder %s: %v", fileName, err)
 			}
 
-			infos[string(normalizedBidderName)] = info
+			bidderInfos[string(normalizedBidderName)] = info
+
+			//need to maintain nullable fields from BidderInfo struct into bidderInfoNullableFields
+			//to handle the default values in aliases yaml
+			if len(info.AliasOf) > 0 {
+				aliasInfo := bidderInfoNullableFields{}
+				if err := yaml.Unmarshal(data, &aliasInfo); err != nil {
+					return nil, fmt.Errorf("error parsing config for aliased bidder %s: %v", fileName, err)
+				}
+
+				aliasInfos[string(normalizedBidderName)] = aliasInfo
+			}
 		}
 	}
-	return infos, nil
+	return processBidderAliases(aliasInfos, bidderInfos)
+}
+
+func processBidderAliases(aliasInfos aliasInfos, bidderInfos BidderInfos) (BidderInfos, error) {
+	for bidderName := range aliasInfos {
+		if err := validateAliases(bidderInfos[bidderName], bidderInfos, bidderName); err != nil {
+			return nil, err
+		}
+		aliasBidderInfo := bidderInfos[bidderName]
+		parentBidderInfo := bidderInfos[aliasBidderInfo.AliasOf]
+		aliasBidderInfo.AppSecret = parentBidderInfo.AppSecret
```

Comment:
So we will always overwrite all these fields of an alias with the value from the parent? I believe we want an alias to specify a different config than the parent for at least some of these.

Reply:
There is a PR dependent on this one for conditional overwrites: #3001

Reply:
That would explain it.